### PR TITLE
Make Swift target main target and remove ObjC target

### PR DESCRIPTION
**Pull request by mdmathias**
This PR removes the existing ObjC target and makes the Swift target the main one. It also renames that Swift target to `GTMAppAuth` and updates `Package.swift`, `GTMAppAuth.podspec`, `Podfile`s, and imports throughout the project to make use of this change.

### CODE REVIEW

File: GTMAppAuth.podspec [hunk 1]
```diff
@@ -1,38 +1,48 @@
 Pod::Spec.new do |s|
 
-  s.name         = "GTMAppAuth"
-  s.version      = "2.0.0"
-  s.summary      = "Authorize GTM Session Fetcher requests with AppAuth via GTMAppAuth"
+  s.name          = "GTMAppAuth"
+  s.version       = "1.3.0"
```
Comment (petea) [inline GTMAppAuth.podspec:4]: Maintain a `'2.0.0'` version string here.

File: GTMAppAuth.podspec [hunk 2]
```diff
@@ -1,38 +1,48 @@
...
+  s.dependency "AppAuth/Core", "~> 1.6"
+
+  s.test_spec 'unit' do |unit_tests|
+    unit_tests.platform = {
+      :ios => ios_deployment_target,
+      :macos => osx_deployment_target,
+    }
+    unit_tests.source_files = [
+      "GTMAppAuth/Tests/Unit/*.swift",
+      "GTMAppAuth/Tests/ObjCIntegration/*.[mh]",
```
Comment (petea) [inline GTMAppAuth.podspec:43]: Should we have the ObjCIntegration tests in their own target as we're doing in the Swift package?

File: GTMAppAuth.podspec [hunk 3]
```diff
@@ -1,38 +1,48 @@
 Pod::Spec.new do |s|
 
-  s.name         = "GTMAppAuth"
-  s.version      = "2.0.0"
-  s.summary      = "Authorize GTM Session Fetcher requests with AppAuth via GTMAppAuth"
+  s.name          = "GTMAppAuth"
```
Comment (petea) [inline GTMAppAuth.podspec:3]: I suggest that we use single quoted strings throughout.

**Comment by petea** [comment]
Don't forget to update the SPM example projects.

File: GTMAppAuth.podspec (hunk 2 above)
Comment (mdmathias) [reply GTMAppAuth.podspec]: Done.

File: GTMAppAuth.podspec [hunk 4]
```diff
@@ -1,38 +1,75 @@
...
+      :watchos => watchos_deployment_target,
+    }
+    unit_tests.source_files = [
+      'GTMAppAuth/Tests/Unit/*.swift',
+      'GTMAppAuth/Tests/Helpers/*.swift',
+    ]
+    unit_tests.dependency 'AppAuth/Core'
+  end
+
+  #s.test_spec 'objc-api-integration' do |api_tests|
```
[the remark this answers is not in the document]
Comment (mdmathias) [reply GTMAppAuth.podspec]: Figured it out!

**Comment by mdmathias** [comment]
@petea I've updated the examples to work with the new library.

**Review by petea — APPROVED** [review]

**Pull request merged**